docs/kyma/04-02-local-installation.md

```diff
@@ -19,88 +20,79 @@ Virtualization:
+  
+  2. Install Kyma on Minikube:
+     ```bash
+     kyma install 
+     ```
+ >**NOTE** If you want to install a specific release version, go to the [GitHub releases page](https://github.com/kyma-project/kyma/releases) to find out more about available releases. Use the release version as a parameter when calling ` kyma install --release {KYMA_RELEASE}`.
+  3. After the installation is completed, you can access the Console UI. Go to [this](https://console.kyma.local) address and select **Login with Email**. Use the **admin@kyma.cx** email address and the password printed in the terminal after installation finished.
+  
+  4. At this point, Kyma is ready for you to explore. See what you can achieve using the Console UI or check out one of the [available examples](https://github.com/kyma-project/examples).
+   
+  </details>
+  <details>
+  <summary>
+  From sources
+  </summary>
+    
+  1. Open a terminal window and navigate to a space in which you want to store local Kyma sources.
+    
+  2. Clone the `Kyma` repository using either HTTPS or SSH. Run:
+     
+     For HTTPS run command:
+     ```bash
+     git clone https://github.com/kyma-project/kyma.git
+     ```
+
+     For SSH run command:     
+     ```bash
+     git clone git@github.com:kyma-project/kyma.git
+     ```
+
+  3. Kyma comes with a local wildcard self-signed `server.crt` certificate. Add this certificate to your OS trusted certificates to access the Console UI. On MacOS, run:
+     ```bash
+     sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain certs/workspace/raw/server.crt
+     ```
+     >**NOTE:** Mozilla Firefox uses its own certificate keychain. If you want to access the Console UI though Firefox, add the Kyma wildcard certificate to the certificate keychain of the browser. To access the Application Connector and connect an external solution to the local deployment of Kyma, you must add the certificate to the trusted certificate storage of your programming environment. Read [this](/components/application-connector#details-access-the-application-connector-on-a-local-kyma-deployment) document to learn more.
+
+  4. Provision a Kubernetes cluster on Minikube. Run:
+     ```bash
+     kyma provision minikube
+     ```
+     >  **NOTE:** The `provision` command uses default Minikube VM driver installed for your OS. For a list of supported VM drivers see [this document](http://github.com/kyma-project/cli).
+  5. Install Kyma from sources. Run:
+     
+     ```bash
+     kyma install --local --src-path {YOUR_KYMA_SOURCE_PATH}
+     ```
+     
+  
+  6. After the installation is completed, you can access the Console UI. Go to [this](https://console.kyma.local) address and select **Login with Email**. Use the **admin@kyma.cx** email address and the password printed in the terminal once the installation process is completed.
```

Comment:
Steps 6 and 7 are universal. Please put them outside of the DIV element. Don't duplicate content unless it's necessary.

Reply:
We will move to general steps section